test/paths-object.test.ts

```diff
@@ -20,7 +20,7 @@ describe("Paths Object", () => {
   test("basic", () => {
     const schema: PathsObject = {
       "/api/v1/user/{user_id}": {
-        parameters: [{ name: "page", in: "query", schema: { type: "number" } }],
+        parameters: [{ name: "page", in: "query", schema: { type: "number" }, description: "Page number." }],
```

Comment:
I'm not sure if I should create a new test instead of updating an existing one

Reply:
Yeah that’s always a tough one and there’s no right answer. I’d say this one is fine here but if we started collecting a lot of these we should break them out into its own test.